rasr/config.py

```diff
@@ -338,3 +341,32 @@ def __init__(self, string, hidden=None):
 
     def __str__(self):
         return self.string
+
+
+class WriteRasrConfigJob(Job):
+    """
+    Write a RasrConfig object into a .config file
+    """
+
+    def __init__(self, config, post_config):
+        """
+        :param RasrConfig config: RASR config part that is hashed
+        :param RasrConfig post_config: RASR config part that is not hashed
+        """
+        self.config = config
+        self.post_config = post_config
+
+        self.out_config = self.output_path("rasr.config")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        self.config._update(self.post_config)
+        with util.uopen(self.out_config, "wt") as f:
+            f.write(repr(self.config))
```

Comment:
Would it be possible/useful/elegant to inherit from `i6_core.rasr.RasrCommand` and use `self.write_config` instead?

At least to make config writing consistent to all other jobs in case the `RasrCommand.write_config` method ever changes.

Reply:
Good hint, will change